lib/interface/cli/commands/hybrid/init.cmd.js

```diff
@@ -77,7 +78,7 @@ function printInstallationOptionsSummary({
     `;
 
     if (appProxy) {
-        summary += `7. App-Proxy hostname: ${colors.cyan(appProxyHost)}\n`;
```

Comment:
just to print the full address in the summary. It's more helpful to us for debugging because that's the value that would actually be used by the ui.